### PR TITLE
added helm examples for port-agent

### PR DESCRIPTION
**Pull request by MatanHeledPort**
# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context.

## Added docs pages

Please also include the path for the added docs

- Quickstart (`/`)
- Blueprint (`/platform-overview/port-components/blueprint`)
- ...

## Updated docs pages

Please also include the path for the updated docs

- Quickstart (`/`)
- Blueprint (`/platform-overview/port-components/blueprint`)
- ...

### CODE REVIEW

File: docs/create-self-service-experiences/setup-backend/webhook/port-execution-agent/Installation.md [hunk 1]
```diff
@@ -79,3 +79,66 @@ Follow one of the guides below:
...
 - [Changelog Listener](/create-self-service-experiences/setup-backend/webhook/examples/changelog-listener.md) - Create a blueprint with `changelogDestination` to listen and act on changes in the software catalog.
 - [GitLab Pipeline Trigger](/create-self-service-experiences/setup-backend/gitlab-pipeline/gitlab-pipeline.md) - Create an action that triggers GitLab Pipeline execution.
+
+
+## Advanced configuration
+Some environments require special configuration when working with the Port agent. This includes working with self-signed certificates and/or proxies.
+
+Port's agent uses Python's [requests](https://requests.readthedocs.io/en/latest/) library. This allows passing advanced configuration using environment variables.
+
+To add an environment variable using the agent's Helm chart, either:
```
Comment (MPTG94) [inline docs/create-self-service-experiences/setup-backend/webhook/port-execution-agent/Installation.md:89]: Here's that either sentence again

File: docs/create-self-service-experiences/setup-backend/webhook/port-execution-agent/Installation.md [hunk 2]
```diff
@@ -79,3 +79,66 @@ Follow one of the guides below:
...
+# The rest of the configuration
+# ...
+env:
+  normal:
+    VAR_NAME: VAR_VALUE
+```
+
+###  Proxy configuration
+
+#### `HTTP_PROXY`,`HTTPS_PROXY` & `ALL_PROXY`
```
Comment (MPTG94) [inline docs/create-self-service-experiences/setup-backend/webhook/port-execution-agent/Installation.md:110]:
```suggestion
#### `HTTP_PROXY`, `HTTPS_PROXY` & `ALL_PROXY`
```

**Review by MPTG94 — APPROVED** [review]
Fix comments and merge

File: docs/create-self-service-experiences/setup-backend/webhook/port-execution-agent/Installation.md (hunk 1 above)
Comment (MatanHeledPort) [reply docs/create-self-service-experiences/setup-backend/webhook/port-execution-agent/Installation.md:89]: WHAT YOU WANT

**Pull request merged**